### PR TITLE
#5153: Group Norm ops in SD model on device. GN fixes.

### PR DESCRIPTION
**Pull request by tt-nshanker**

### CODE REVIEW

**Comment by tt-nshanker** [comment]
Cleaning up in progress. All prints to screen will be removed.

File: ttnn/ttnn/operations/normalization.py [hunk 1]
```diff
@@ -217,7 +286,7 @@ def _torch_group_norm(input_tensor: ttnn.Tensor, *, num_groups, epsilon=1e-05, w
         bias = ttnn.to_torch(bias)
         if len(bias.shape) == 2:
             bias = bias[0]
-
+    print("Starting gn functional torch")
     return torch.nn.functional.group_norm(input_tensor, num_groups, weight, bias, eps=epsilon)
```
Comment (eyonland) [inline ttnn/ttnn/operations/normalization.py:290]: did you mean to leave the print statement?

**Review by eyonland — APPROVED** [review]

**Review by mywoodstock — APPROVED** [review]
